src/components/common/Table/Table.component.tsx

```diff
@@ -386,33 +396,44 @@ const Table = <T extends object>({
                             )
                           : getOwnValueByKey(row, accessor as any);
 
+                        const normalContents = renderCustomCell
+                          ? renderCustomCell(cellValue, id, undefined, applicationParams)
+                          : cellValue;
+
+                        const blockContents = renderCustomCell
+                          ? renderCustomCell(cellValue, id, () => {
+                              handleAddToast({
+                                type: ToastType.error,
+                                message: '접근 권한이 없는 팀입니다.',
```

Comment:
매쉬업에서 팀은 프로젝트를 지칭하고 플랫폼이 안드, 아오스, 웹 등등을 의미하고 있어서 워딩을 이렇게 바꿔주는게 좋을거 같아요!
```suggestion
                                message: '접근 권한이 없는 플랫폼 입니다.',
```

Reply:
디자인 시안을 바꿔야겠네요 👍